products/spanner/inspec.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  InstanceConfig: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: true
+  Instance: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: false
+    properties:
+      state: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true  
+  Database: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: false
+    properties:
```

Comment:
Toss on a plural_exclude for the `instance` field, it doesn't come back on the response:

     instance: !ruby/object:Overrides::Inspec::PropertyOverride
         exclude_plural: true

Reply:
Which properties , we need to exclude?

Reply:
The `instance` property needs `exclude_plural` because it is not included in the API list response.

Reply:
I dont see "instance" is property in api.yaml for Database resource. It is defined as parameter only.